demos/phi-3-mini/llm.js

```diff
@@ -79,7 +79,9 @@ export class LLM {
             ],
         };
 
-        log(`WebNN EP config: ${model} · ${this.dtype} · ${session_options.executionProviders[0].name} · ${session_options.executionProviders[0].deviceType}`);
+        log(
```

Comment:
Hmm, ESLint makes some of these less readable (like this, where obviously keeping it altogether on a single line would be clearer to read). Oh well.